authentication/migrations/0005_authuser_alter_customuser_user_id.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 4.1 on 2024-02-21 10:56
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('authentication', '0004_authuser_companies_remove_customuser_id_and_more'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='AuthUser',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('last_login', models.DateTimeField(blank=True, null=True, verbose_name='last login')),
+                ('email', models.EmailField(default=None, max_length=100, unique=True)),
+                ('password', models.CharField(default=None, max_length=128)),
+                ('first_name', models.CharField(default=None, max_length=100)),
+                ('surname', models.CharField(default=None, max_length=100)),
+                ('position', models.CharField(default=None, max_length=25)),
+                ('company_id', models.IntegerField(default=None)),
+                ('user_id', models.IntegerField(default=None)),
+                ('is_startup', models.BooleanField(default='undefined')),
+                ('is_verified', models.BooleanField(default='undefined')),
+                ('is_superuser', models.BooleanField(default=False)),
+                ('is_authenticated', models.BooleanField(default=False)),
+                ('error', models.CharField(default=None, max_length=150)),
```

Comment:
Since AuthUser model created solely for authentication purpose and serves as blueprint for required user and company information, I decided to use it also for error handling: if any error occures, the empty AuthUser instance is returned with error stored in the error field.